index.src.html

```diff
@@ -2129,44 +2143,161 @@
 
 : <dfn enum-value for=AudioSampleFormat>FLTP</dfn>
-:: 32-bit float samples with planar channel arrangement.
+:: [[WEBIDL#idl-float|32-bit float]] samples with [=planar=] channel arrangement.
+
+
+### Arrangement of audio buffer ### {#audio-buffer-arrangement}
+
+When an {{AudioData}} has an {{AudioSampleFormat}} that is
+<dfn>interleaved</dfn>, the audio samples from different channels are laid out
+consecutively in the same buffer, in the order described in the section
+[[#audio-channel-ordering]]. The {{AudioData}} has a single plane, that contains a
+number of elements therefore equal to {{AudioData/numberOfFrames}} *
+{{AudioData/numberOfChannels}}.
+
+When an {{AudioData}} has an {{AudioSampleFormat}} that is
+<dfn>planar</dfn>, the audio samples from different channels are laid out
+in different buffers, themselves arranged in an order described in the section
+[[#audio-channel-ordering]]. The {{AudioData}} has a number of planes equal to the
+{{AudioData}}'s {{AudioData/numberOfChannels}}. Each plane contains
+{{AudioData/numberOfFrames}} elements.
+
+Note: The [[WEBAUDIO|Web Audio API]] currently uses {{FLTP}} exclusively.
+
+### Magnitude of the audio samples ### {#audio-samples-magnitude}
+
+The <dfn>minimum value</dfn> and <dfn>maximum value</dfn> of an audio sample,
+for a particular audio sample type, are the value below which
+(respectively above which) audio clipping might occur. They are otherwise regular
+types, that can hold values outside this interval, during intermeditate
+processing.
+
+The <dfn>bias value</dfn> for an audio sample type is the value that often
+corresponds to the middle of the range (but often the range is not symmetrical).
+An audio buffer comprised only of values equal to the [=bias value=] is silent.
+
+<table id="sample-types">
+<thead>
+<tr class="header">
+<th>[=Audio sample type|Type=]</th>
+<th>Type</th>
+<th>[=Minimum value=]</th>
+<th>[=Bias value=]</th>
+<th>[=Maximum value=]</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>{{U8}}</td>
+<td>[[WEBIDL#idl-octet|octet]]</td>
+<td>0</td>
+<td>128</td>
+<td>+255</td>
+</tr>
+<tr class="even">
+<td>{{S16}}</td>
+<td>[[WEBIDL#idl-short|short]]</td>
+<td>-32768</td>
+<td>0</td>
+<td>+32767</td>
+</tr>
+<tr class="odd">
+<td>{{S24}}</td>
+<td>[[WEBIDL#idl-long|long]]</td>
+<td>-8388608</td>
+<td>0</td>
+<td>+8388607</td>
+</tr>
+<tr class="even">
+<td>{{S32}}</td>
+<td>[[WEBIDL#idl-long|long]]</td>
+<td>-2147483648</td>
+<td>0</td>
+<td>+2147483647</td>
+</tr>
+<tr class="odd">
+<td>{{FLT}}</td>
+<td>[[WEBIDL#idl-float|float]]</td>
+<td>-1.0</td>
+<td>0.0</td>
+<td>+1.0</td>
+</tr>
+</tbody>
+</table>
+
+Note: There is no data types that can hold 24 bits of information conveniently,
+but audio content using 24-bits samples is common, so 32-bits integers are
+commonly used to hold 24-bits content.
+
+### Audio channel ordering ### {#audio-channel-ordering}
+
+When decoding, the ordering of the audio channels in an {{AudioData}} MUST be
```

Comment:
I like how you used 'resulting' in the encoding rule. Suggest the following edit:

> When decoding, the ordering of the audio channels in the resulting AudioData MUST be the same as what is present in the given EncodedAudioChunk.

For the encoding rule, I would phrase closer to what you have for decoding... eg.

> When encoding, the ordering of the audio channels in the resulting EncodedAudioChunk MUST be the same as what is present in the given AudioData.